### ISSUE TITLE
Cannot send encrypted emails

### ISSUE THREAD

**Issue by flo1o1** [thunderbird/thunderbird-android#3713]
I use K9 and Openkeychain installed. K9 and Openkeychain are linked to each other, and I selected the keys which should be available to k9. Both PGP keys have subkeys for signing and encryption. 

When I compose a message, enable encryption and submit, then I get an error "send failed: null" 

- Android 8.1.0
- K-9 Mail 5.600
- Openkeychain 5.2 (52009)

- multiple mail accounts
- multiple pgp keys (newly created in Openkeychain) 

Any hints? 

Best, 
Florian

**Comment by dennis-mb** [comment]
I ran into the exact same "send fail: null" error and found this issue. At least in my case, i forgot to update my key with a new sign-only subkey. So my setting was different to that of @floeschie and not a bug. With signing subkey present, everything works fine for me.

But as screenshots in #2175 show that at least in earlier versions, there was a well understandable error stating the fact of a missing signing subkey. (Plus one knowing, that encrypt-only is by design #1961).

I filed that as RFE in #3949.
